lib/main.js

```diff
@@ -1,81 +1,191 @@
 'use babel'
 
-import {CompositeDisposable} from 'atom'
-import GoInformationView from './components/go-information-view'
-import {GoInformation} from './go-information'
-import {PanelManager} from './panel-manager'
-import {Formatter} from './formatter'
-import {Tester} from './tester'
-import TestPanel from './components/test-panel'
-import {TestPanelManager} from './test-panel-manager'
-import {ToolChecker} from './tool-checker'
+import {CompositeDisposable, Disposable} from 'atom'
+
+const oldPackages = [
+  'gofmt',
+  'tester-go',
+  'builder-go',
+  'autocomplete-go',
+  'godoc',
+  'gorename',
+  'go-hyperclick',
+  'navigator-go',
+  'go-get',
```

Comment:
No, go-get and go-config will be end of life once this PR merges. Packages wishing to make use of that behavior will use go-plus as their dependency, as it offers those same services.